### PR TITLE
Fix issue #2003

### PR DESCRIPTION
**Pull request by JieJhih**

### CODE REVIEW

File: internal/pkg/config/bgp_configs.go [hunk 1]
```diff
@@ -1416,7 +1416,9 @@ type BmpServerConfig struct {
 	// gobgp:route-mirroring-enabled's original type is boolean.
 	// Enable feature for mirroring of received BGP messages
 	// mainly for debugging purpose.
-	RouteMirroringEnabled bool `mapstructure:"route-mirroring-enabled" json:"route-mirroring-enabled,omitempty"`
+	RouteMirroringEnabled bool   `mapstructure:"route-mirroring-enabled" json:"route-mirroring-enabled,omitempty"`
+	SysName               string `mapstructure:"SysName" json:"SysName,omitempty"`
+	SysDescr              string `mapstructure:"SysDescr" json:"SysDescr,omitempty"`
 }
```
Comment (fujita) [inline internal/pkg/config/bgp_configs.go]: bgp_configs.go is automatically generated. you can modify it directly
https://github.com/osrg/gobgp/tree/master/tools/pyang_plugins

File: internal/pkg/config/default.go [hunk 2]
```diff
@@ -350,6 +350,9 @@ func setDefaultConfigValuesWithViper(v *viper.Viper, b *BgpConfigSet) error {
 	}
 
 	for idx, server := range b.BmpServers {
+		if server.Config.SysName == "" || server.Config.SysDescr == "" {
+			return fmt.Errorf("BMP Initiation Message must contain sysDescr and sysName Information TLVs")
+		}
```
Comment (fujita) [inline internal/pkg/config/default.go]: What are appropriate default strings for them? I prefer to set something appropriate if not specified.

File: internal/pkg/config/bgp_configs.go (hunk 1 above)
Comment (fujita) [reply internal/pkg/config/bgp_configs.go]: let me know if you like me to take care of changes to bgp_configs.go.

Comment (JieJhih) [reply internal/pkg/config/bgp_configs.go]: Thanks, I fixed it, and I also fixed the pyang plugins document. Is there have something error please help me fix it out.

File: internal/pkg/config/default.go (hunk 2 above)
Comment (JieJhih) [reply internal/pkg/config/default.go]: RFC1213
`sysDescr OBJECT-TYPE
              SYNTAX  DisplayString (SIZE (0..255))
              ACCESS  read-only
              STATUS  mandatory
  DESCRIPTION
                      "A textual description of the entity.  This value
                      should include the full name and version
                      identification of the system's hardware type,
                      software operating-system, and networking
                      software.  It is mandatory that this only contain
                      printable ASCII characters."`
`sysName OBJECT-TYPE
              SYNTAX  DisplayString (SIZE (0..255))
 ACCESS  read-write
              STATUS  mandatory
              DESCRIPTION
                      "An administratively-assigned name for this
                      managed node.  By convention, this is the node's
                      fully-qualified domain name."`

Comment (JieJhih) [reply internal/pkg/config/default.go]: I think SysName could have the default value, but I'm not sure SysDescr .

File: cmd/gobgpd/main.go [hunk 3]
```diff
@@ -28,7 +28,7 @@ import (
 
 	"github.com/coreos/go-systemd/daemon"
 	"github.com/golang/protobuf/ptypes/any"
-	"github.com/jessevdk/go-flags"
```
Comment (fujita) [inline cmd/gobgpd/main.go]: why?

Comment (JieJhih) [reply cmd/gobgpd/main.go]: Oops , I'm sorry, I didn't noticed that. This caused by my vscode plugin.
I fixed it.

**Comment by fujita** [comment]
thanks, squashed and pushed.

**Pull request closed without merging**